app/fissile.go

```diff
@@ -646,19 +655,12 @@ func getDiffsFromReleases(releases []*model.Release) (*HashDiffs, error) {
 	for idx, release := range releases {
 		configs := release.GetUniqueConfigs()
 		for _, config := range configs {
-			key, err := configstore.BoshKeyToConsulPath(config.Name, configstore.DescriptionsStore)
-			if err != nil {
-				return nil, fmt.Errorf("Error getting config %s for release %s: %s", config.Name, release.Name, err.Error())
-			}
-			hashes[idx][key] = config.Description
+			hashes[idx][config.Name] = config.Description
```

Comment:
I don't see how that would happen. Each config.name indexes into a release-based slice of the `hashes`